assets/src/components/detours/detourPanels/detourFinishedPanel.tsx

```diff
@@ -81,15 +83,26 @@ export const DetourFinishedPanel = ({
       </Panel.Body.ScrollArea>
 
       <Panel.Body.Footer className="d-flex flex-column">
-        {onActivateDetour && (
-          <Button
-            className="m-3 flex-grow-1 icon-link justify-content-center"
-            onClick={onActivateDetour}
-            data-fs-element="Begin Activate Detour"
-          >
-            <BsIcons.Power />
-            Start Detour
-          </Button>
+        {onDeleteDetour && (
```

Comment:
The `onActivateDetour` check has been lost, so users who do have "activation" permissions will lose access to the activation button if they don't have the "deletion" permissions